README.md

```diff
@@ -1,40 +1,95 @@
 
-[![npm][npm-image]][npm-url]
-[![travis][travis-image]][travis-url]
-
-[npm-image]: https://img.shields.io/npm/v/eslint-config-semistandard.svg?style=flat-square
-[npm-url]: https://www.npmjs.com/package/eslint-config-semistandard
-[travis-image]: https://img.shields.io/travis/standard/eslint-config-semistandard.svg?style=flat-square
+[travis-image]: https://img.shields.io/travis/standard/eslint-config-semistandard/master.svg
 [travis-url]: https://travis-ci.org/standard/eslint-config-semistandard
+[npm-image]: https://img.shields.io/npm/v/eslint-config-semistandard.svg
+[npm-url]: https://npmjs.org/package/eslint-config-semistandard
+[downloads-image]: https://img.shields.io/npm/dm/eslint-config-semistandard.svg
+[downloads-url]: https://npmjs.org/package/eslint-config-semistandard
+[semistandard-badge]: https://raw.githubusercontent.com/standard/semistandard/master/badge.svg
+[semistandard-image]: https://img.shields.io/badge/code_style-semistandard-brightgreen.svg
+[semistandard-url]: https://github.com/standard/semistandard
+[standard-url]: https://standardjs.com
+
+#### An ESLint [Shareable Config](http://eslint.org/docs/developer-guide/shareable-configs) for [JavaScript Semistandard Style][semistandard-url]
 
-eslint sharable config for semistandard – like [eslint-config-standard](https://github.com/feross/eslint-config-standard), but with semicolons
+[![JavaScript Style Guide - Standard Style][semistandard-badge]][semistandard-url]
 
 ## Install
 
-Use [`install-peerdeps`](https://npm.im/install-peerdeps):
+This module is for advanced users. You probably want to use [`semistandard`][semistandard-url] instead :)
 
+```bash
+npm install eslint-config-semistandard
 ```
-npx install-peerdeps --dev eslint-config-semistandard
+
+## Usage
+
+Shareable configs are designed to work with the `extends` feature of `.eslintrc` files.
+You can learn more about
+[Shareable Configs](http://eslint.org/docs/developer-guide/shareable-configs) on the
+official ESLint website.
+
+**If you want to set up the config automatically**, follow these steps in your project directory:
+
+1. `npx eslint --init`
+2. Select "Use a popular style guide."
+3. Select "Standard."
+4. Select a config file format.
+5. If prompted, confirm the installation of the necessary dependencies.
+6. `npm install eslint-config-semistandard`
+7. In your config file, change `extends: 'standard'` to `extends: 'semistandard'`.
+
+The above steps will automatically set up an ESLint configuration and install the necessary dependencies for you.
+
+**If you want to set up the config manually**, run the following command:
+
+```bash
+npm install --save-dev eslint-config-semistandard eslint-config-standard eslint-plugin-promise eslint-plugin-import eslint-plugin-node
 ```
-or the classic way:
+
+Then, add this to your `.eslintrc` file:
```

Comment:
It is a smart thing to refrain from suggesting a particular configuration format. I don't know what is ESLint's folks preferred format these days but I'm sure it is really their job to push forward either `.eslintrc` or `.eslintrc.js`, and not something that should be advertised here.